org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -58,32 +63,38 @@ public InitHandler(ProjectsManager manager, PreferenceManager preferenceManager,
 		this.preferenceManager = preferenceManager;
 	}
 
-
-	InitializeResult initialize(InitializeParams param){
-		logInfo("Initializing Java Language Server "+JavaLanguageServerPlugin.getVersion());
-		if(param.getCapabilities()==null){
+	InitializeResult initialize(InitializeParams param) {
+		logInfo("Initializing Java Language Server " + JavaLanguageServerPlugin.getVersion());
+		if (param.getCapabilities() == null) {
 			preferenceManager.updateClientPrefences(new ClientCapabilities());
-		}else{
+		} else {
 			preferenceManager.updateClientPrefences(param.getCapabilities());
 		}
 		String rootPath = param.getRootUri();
 		if (rootPath != null) {
 			URI uri = URI.create(rootPath);
-			if ("file".equals(uri.getScheme())){
+			if ("file".equals(uri.getScheme())) {
 				rootPath = Paths.get(uri).toString();
 			}
 		}
 		if (rootPath == null) {
 			rootPath = ResourcesPlugin.getWorkspace().getRoot().getLocation().toOSString();
-			logInfo("No root uri was defined. Falling back on "+rootPath);
+			logInfo("No root uri was defined. Falling back on " + rootPath);
 		}
 		triggerInitialization(rootPath);
 		ResourcesPlugin.getWorkspace().addResourceChangeListener(new WorkspaceDiagnosticsHandler(connection, projectsManager), IResourceChangeEvent.POST_BUILD | IResourceChangeEvent.POST_CHANGE);
 		JavaLanguageServerPlugin.getLanguageServer().setParentProcessId(param.getProcessId().longValue());
+		try {
+			List<String> bundleList = getBundleList(param.getInitializationOptions());
+			BundleUtils.loadBundles(bundleList);
+		} catch (CoreException e) {
+			// The additional plug-ins should not affect the main language server loading.
+			JavaLanguageServerPlugin.logException("Failed to load extension bundles ", e);
+		}
 		InitializeResult result = new InitializeResult();
 		ServerCapabilities capabilities = new ServerCapabilities();
 		capabilities.setTextDocumentSync(TextDocumentSyncKind.Incremental);
-		capabilities.setCompletionProvider(new CompletionOptions(Boolean.TRUE, Arrays.asList(".","@","#")));
+		capabilities.setCompletionProvider(new CompletionOptions(Boolean.TRUE, Arrays.asList(".", "@", "#")));
```

Comment:
I've noticed a couple of formatting changes. Can we coordinate on the formatter settings in order to reduce those?